packages/validators/src/server/external/index.ts

```diff
@@ -13,7 +13,8 @@ export const RemoveFriendSchema = z.object({
 });
 
 export const CreateReviewSchema = z.object({
-  restaurantId: z.number().int().positive(),
+  restaurantId: z.string(),
+  name: z.string(),
```

Comment:
```suggestion
  restaurantName: z.string(),
```
Try and restrict the value here as well (min and max lengths)

Reply:
done